lib/objectBuilder.js

```diff
@@ -110,11 +110,23 @@ module.exports = function () {
   }
 
   function makeSMBG() {
-    return _makeWithValue('smbg');
+    var rec = _.assign(_createObject(), deviceInfo, {
+      type: 'smbg',
+      value: REQUIRED,
+      units: REQUIRED
+    });
+    rec._bindProps();
+    return rec;
   }
 
   function makeCBG() {
-    return _makeWithValue('cbg');
+    var rec = _.assign(_createObject(), deviceInfo, {
+      type: 'cbg',
+      value: REQUIRED,
+      units: REQUIRED
+    });
+    rec._bindProps();
+    return rec;
   }
 
   function makeNote() {
```

Comment:
Now that _makeWithValue is used in only one place, could you also just remove it from makeNote and eliminate the function?